tfhe/c_api_tests/test_high_level_zk.c

```diff
@@ -35,6 +35,11 @@ int main(void) {
   status = compact_pke_crs_public_params(crs, &public_params);
   assert(status == 0);
 
+  // This is a hack to have metadata_len be "constant" i.e. cannot change between compilation and
+  // runtime, otherwise metadata[metadata_len] is considered a Variable Length Array
+  enum { metadata_len = (size_t)5 };
```

Comment:
how about `#define METADATA_LEN 5` ?

Reply:
\*sigh\*

why can't we have nice things, macros for array len 😭